doc/pachub/pachub_getting_started.md

```diff
@@ -55,17 +55,18 @@ a limited time.
 To create a Pachyderm cluster, complete the following steps:
 
 1. If you have not yet done so, log in to Pachyderm Hub.
-1. Click **Create**.
+1. Click **Create cluster**.
 1. Type a name for your cluster. For example, `test1`.
 1. Click **Create**.
 
    Your cluster is provisioned instantly!
 
    ![Pachub cluster](../images/s_pachub_cluster.png)
 
-   **Note:** Pachyderm has a set number of pre-warmed clusters.
-   If you see your cluster is in a *starting* state, you might
-   have to wait a few minutes for it to be ready.
+   **Note:** While Pachyderm maintains a few clusters that are instantly
+   available, none may be available during periods of high traffic. If
+   you see your cluster is in a *starting* state, you might have to wait a few
+   minutes for it to be ready.
```

Comment:
This was changed because I don't think most users will know what a pre-warmed cluster means